target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -10,10 +10,20 @@ board=$(mvebu_board_name)
 case "$board" in
 armada-xp-linksys-mamba)
         # Set one execution of the fan script during every start (waiting for cron)
-        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
+        if grep -q fan_ctrl.sh /etc/rc.local
+                then
+                        exit 0
+                else
+                        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
```

Comment:
The /etc/rc.local file is user property, with this code you would overwrite it on the first boot.